test/cpp_extensions/setup.py

```diff
@@ -16,7 +21,8 @@
             'cuda_extension_kernel2.cu',
         ],
         extra_compile_args={'cxx': ['-g'],
-                            'nvcc': ['-O2']})
+                            'nvcc': ['-O2']},
+        extra_link_args=['cudart.lib'] if IS_WINDOWS else [])
```

Comment:
Should this extra link be present in the default `cpp_extensions` instead of in here?

Reply:
What does it mean to present in the default cpp_extensions`?

Reply:
I think we should link by default to `cudart` if we are compiling cuda extensions. That's the case for Linux I believe, so we should follow the same pattern I think